Tools/clinic/clinic.py

```diff
@@ -2209,21 +2252,36 @@ def print_block(
         write("\n")
 
         output = ''
-        if core_includes:
-            if not limited_capi:
-                output += textwrap.dedent("""
-                    #if defined(Py_BUILD_CORE) && !defined(Py_BUILD_CORE_MODULE)
-                    #  include "pycore_gc.h"            // PyGC_Head
-                    #  include "pycore_runtime.h"       // _Py_ID()
-                    #endif
-
-                """)
-
+        if core_includes and header_includes:
             # Emit optional "#include" directives for C headers
-            for include, reason in sorted(header_includes.items()):
-                line = f'#include "{include}"'.ljust(35) + f'// {reason}\n'
+            output += '\n'
+
+            def sort_key(include: Include) -> tuple[str, str]:
```

Comment:
This could be made a method on the `Include` class, and then line 2264 could be changed to

```py
includes = sorted(header_includes.values(), key=Include.sort_key)
```